### ISSUE TITLE
Utilize arithmetics of Complex numbers

### ISSUE THREAD

**Issue by tobolar** [modelica/Modelica_LinearSystems2#97]
Elementary arithmetics given by "Complex" package shall be used instead of corresponding operators from `M_LS2.Math.Complex`, e.g. `M_LS2.Math.Complex.'+'` -> `Complex.'+'` or `M_LS2.Math.Complex.'*'` -> `Complex.'*'`. This could be reached by appropriate usage of Complex within concerned classes.

**Comment by tobolar** [comment]
@HansOlsson Any possibility to convert the used function `Modelica_LinearSystems2.Math.Complex.j` into the constant `Modelica.ComplexMath.j`? I.e. the declaration
```
  Complex j = Modelica_LinearSystems2.Math.Complex.j():
```
shall be converted to 
```
  Complex j = Modelica.ComplexMath.j;
```
<hr>

Using simple class conversion
```
convertClass("Modelica_LinearSystems2.Math.Complex.j", "Modelica.ComplexMath.j");
```
fails.

**Comment by tobolar** [comment]
@HansOlsson Kind reminder.

**Comment by tobolar** [comment]
This shall be resolved with a package collecting obsolete classes which can not be converted.

**Comment by tobolar** [comment]
Thanks for clearifying. I can consider this in the documentation.
Btw. the corresponding annotation is called "obsolete" indeed. ;-)

**Comment by dietmarw** [comment]
I know :-(

**Comment by tobolar** [comment]
Resolved with 090e6c20ec173708245cb00740f3c5410bb14a5d (package ObsoleteLinearSystems2) and 3f3d415ee3910f593ebe98d7a2eba289d8d90ac0 (conversion script).
